$MirOS: ports/print/cups/patches/patch-cups_Makefile,v 1.1.7.1 2005/03/18 15:51:00 tg Exp $
--- cups/Makefile.orig	Thu May  4 20:53:12 2006
+++ cups/Makefile	Mon Jun 19 19:48:31 2006
@@ -161,9 +161,9 @@ depend:
 install:	all installhdrs $(INSTALLSTATIC) $(INSTALL32) $(INSTALL64)
 	$(INSTALL_DIR) -m 755 $(LIBDIR)
 	$(INSTALL_LIB) $(LIBCUPS) $(LIBDIR)
-	if test $(LIBCUPS) = "libcups.so.2" -o $(LIBCUPS) = "libcups.sl.2"; then \
-		$(RM) $(LIBDIR)/`basename $(LIBCUPS) .2`; \
-		$(LN) $(LIBCUPS) $(LIBDIR)/`basename $(LIBCUPS) .2`; \
+	if test $(LIBCUPS) = "libcups.so.2.2" -o $(LIBCUPS) = "libcups.sl.2.2"; then \
+		$(RM) $(LIBDIR)/`basename $(LIBCUPS) .2.2`; \
+		$(LN) $(LIBCUPS) $(LIBDIR)/`basename $(LIBCUPS) .2.2`; \
 	fi
 	if test $(LIBCUPS) = "libcups.2.dylib"; then \
 		$(STRIP) -x $(LIBDIR)/$(LIBCUPS); \
@@ -284,7 +284,7 @@ libcups_s.a:	$(LIBOBJS) libcups_s.exp
 
 libcups.la:    $(LIBOBJS)
 	echo Linking $@...
-	$(CC) $(ARCHFLAGS) $(DSOFLAGS) -o $@ $(LIBOBJS:.o=.lo) -rpath $(LIBDIR) \
+	$(DSO) $(ARCHFLAGS) $(DSOFLAGS) -o $@ $(LIBOBJS:.o=.lo) -rpath $(LIBDIR) \
 		-version-info 2:7 $(SSLLIBS) $(COMMONLIBS) $(LIBZ)
 
 
